libpod/image/image_test.go

```diff
@@ -0,0 +1,184 @@
+package image
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path"
+	"testing"
+
+	"github.com/containers/storage"
+	"github.com/containers/storage/pkg/reexec"
+	"github.com/pkg/errors"
+	"github.com/projectatomic/libpod/libpod"
+	"github.com/stretchr/testify/assert"
+)
+
+var (
+	bbNames      = []string{"docker.io/library/busybox:latest", "docker.io/library/busybox", "docker.io/busybox:latest", "docker.io/busybox", "busybox:latest", "busybox"}
+	bbGlibcNames = []string{"docker.io/library/busybox:glibc", "docker.io/busybox:glibc", "busybox:glibc"}
+	fedoraNames  = []string{"registry.fedoraproject.org/fedora-minimal:latest", "registry.fedoraproject.org/fedora-minimal", "fedora-minimal:latest", "fedora-minimal"}
+)
+
```

Comment:
It would probably be good to have a test that does a commit in here somewhere.

Reply:
Commit should probably be unit tested elsewhere, since it's a container operation and this is the image package. I'd like it if this was eventually able to be used completely separate from libpod

Reply:
I think these are probably covered in the tests for the commands themselves, but you might consider testing inspect, load, save and rmi here at least a little bit.  However if you think the functionality is well covered in other tests, then no worries.

Reply:
@TomSweeneyRedHat stage 1 is just about resolving the local and remote names correctly.  All of that comes in due time.